src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const add = (a) => {
+    if (a === undefined) {
```

Comment:
The check for `a === undefined` is correct, but ensure that this logic is intended for when no arguments are passed to the `add` function. This is crucial for the function to reset the sum and return it.